### PR TITLE
updated for changes at wwdc 2016

### PR DESCRIPTION
**Pull request by chilcote**
Hopefully the element changes will iron out before their final release.  Otherwise, I'm sure there's a cleaner way to do this.

### CODE REVIEW

File: createOSXinstallPkg [hunk 1]
```diff
@@ -547,6 +552,12 @@ def getPkgAndMakeIndexSproduct(destpath, os_vers='10.7'):
         'index-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard'
         '.merged-1.sucatalog')
 
+    SIERRA_PKGNAME = 'OSX_10_12_IncompatibleAppList.pkg'
+    SIERRA_CATALOG_URL = (
+        'https://swscan.apple.com/content/catalogs/others/'
+        'index-10.12seed-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard'
```
Comment (gregneagle) [inline createOSXinstallPkg]: Looks like the pkg we want is also in the "production" 10.12 catalog: https://swscan.apple.com/content/catalogs/others/index-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog -- I think we should use that Catalog URL instead.

File: createOSXinstallPkg [hunk 2]
```diff
@@ -298,7 +300,10 @@ def getItemsFromDist(filename):
         item_dict['title'] = title_elements[0].firstChild.wholeText
     script_elements = dom.getElementsByTagName('script')
     if len(script_elements):
-        item_dict['script'] = script_elements[0].toprettyxml()
+        if os_version == '10.12':
+            item_dict['script'] = script_elements[1].toprettyxml()
```
Comment (gregneagle) [inline createOSXinstallPkg]: Interesting. I don't know if I've ever seen a Distribution file with multiple script elements...

**Comment by chilcote** [comment]
Hope I did that right...

**Comment by gregneagle** [comment]
Looks good to me...

**Pull request merged**